src/cider/nrepl/middleware/stacktrace.clj

```diff
@@ -8,9 +8,14 @@
             [clojure.tools.nrepl.middleware :refer [set-descriptor!]]
             [clojure.tools.nrepl.middleware.session :refer [session]]
             [clojure.tools.nrepl.misc :refer [response-for]]
-            [clojure.tools.nrepl.transport :as t])
+            [clojure.tools.nrepl.transport :as t]
+            [clojure.java.io :as io]
+            [clojure.tools.namespace.find :as nsfind])
   (:import (clojure.lang Compiler$CompilerException)))
 
+(def project-name
+  (System/getProperty "user.dir"))
```

Comment:
an aborted first attempt that was accidentally committed. it is removed in a later commit and will disappear in the squash.